schema.graphql

```diff
@@ -106,7 +106,7 @@ type Contract @entity {
 type LegacyBridgeSwap @entity {
   id: ID! # id field is always required and must look like this
   destination: String!
-  contract: String!
+  contract: Contract! @index
```

Comment:
I don't think we need to use the `@index` decorator on relationships because the foreign key column will get an index automatically. I checked the table definition for `events` (which doesn't do this) and it seems to have an index on both the transaction and block foreign keys (IDs):

![image](https://user-images.githubusercontent.com/600733/206673860-4b24e8bc-da3e-49b3-bd6e-ac21e1874c82.png)